README.md

```diff
@@ -11,7 +11,9 @@ For in-depth information on the Graph Visualizer, see this tutorial: [TensorBoar
 
 You may also want to watch
 [this video tutorial](https://www.youtube.com/watch?v=eBbEDRsCmv4) that walks
-through setting up and using TensorBoard.
+through setting up and using TensorBoard. There's an associated [tutorial with
```

Comment:
Consider having only "associated tutorial" be the hyperlink. It makes more sense from an accessibility and aesthetics perspective IMHO.